monkey/monkey_island/cc/server_utils/encryption/data_store_encryptor.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+# PyCrypto is deprecated, but we use pycryptodome, which uses the exact same imports but
+# is maintained.
+from Crypto import Random  # noqa: DUO133  # nosec: B413
+
+from monkey_island.cc.server_utils.encryption.key_based_encryptor import KeyBasedEncryptor
```

Comment:
If we import `KeyBasedEncryptor` in `encryption/__init__.py`, then this import becomes:
```suggestion
from monkey_island.cc.server_utils.encryption import KeyBasedEncryptor
```